docs/release_process.md

```diff
@@ -134,3 +135,20 @@ Once the release is out:
 1. Forward-port the CHANGELOG.md changes from the release branch to master. This
    includes the release date and the entries which were backported from master
    after the release branch was cut.
+1. Create a "Release Notes" page on the bpftrace website ([example](https://bpftrace.org/release-024-notes))
+   and update the text and link on the homepage to point to this new page.
+   Also link to this page from the "Release Highlights" section in the tag
+   description.
+
+## Creating a Patch Release
+
+1. Checkout the active release branch.
+1. Mark the release in [CHANGELOG.md](../CHANGELOG.md) by replacing the `##
+   Unreleased` header with `## [0.0.Z] TODAY'S DATE` or adding this header
```

Comment:
Shouldn't be `[0.0.Z]` but something like `[X.Y.Z]`, right?

Reply:
Yeah that's better. Will update.